scripts/circle-ci-notifications.coffee

```diff
@@ -0,0 +1,26 @@
+watchers = {
+  "besu": [
+      {regex: /master/, channels: ["besu-contributors"]},
+      {regex: /release-.*/, channels: ["besu-contributors"]}
```

Comment:
Would be worth creating a default_channel="besu-contributors" and using that var

Reply:
I have added f923fc2caeec2904de652c5622c352214f20b78f as an alternative to multiple channels for a build branch regex.  Does that fit better with your idea?

Reply:
:+1: